packages/wallet-frontend/src/components/kyc/PersonalDetailsForm.tsx

```diff
@@ -100,6 +100,8 @@ export const PersonalDetailsForm = () => {
         name="country"
         render={({ field: { value } }) => (
           <Select<SelectOption>
+            required
+            label="Country"
             options={countries}
```

Comment:
Should have maybe a placeholder of Select country... to be consistent with other selects